mono_repo/lib/src/utils.dart

```diff
@@ -65,18 +43,6 @@ List<String> listPackageDirectories(
               ' an expected `$pubspecFileName` in `$relativeSubDirPath`.');
         }
 
-        var pubspecContent = y.loadYaml(pubspecFile.readAsStringSync()) as Map;
```

Comment:
Also removed the pubspec validation here. Will do an update that centralizes logic using `pubspec_parse`